src/unpack_and_setup.sh

```diff
@@ -114,14 +114,16 @@ if [ -e ${TempSubjectDir}/DCMs/${SUB}/${VISIT}/dwi ]; then
                 echo "Replacing GE bvals and bvecs for software version after DV25 and before DV29"
                 cp `dirname $0`/ABCD_Release_2.0_Diffusion_Tables/GE_bvals_DV26.txt ${orig_bval}
                 cp `dirname $0`/ABCD_Release_2.0_Diffusion_Tables/GE_bvecs_DV26.txt ${orig_bvec}
-            elif [[ `dcmdump --search 0008,0070 ${first_dcm} 2>/dev/null` == *SIEMENS* ]]; then
-                echo "Replacing Siemens bvals and bvecs"
-                cp `dirname $0`/ABCD_Release_2.0_Diffusion_Tables/Siemens_bvals.txt ${orig_bval}
-                cp `dirname $0`/ABCD_Release_2.0_Diffusion_Tables/Siemens_bvecs.txt ${orig_bvec}
-            else
-                echo "ERROR setting up DWI: Manufacturer not recognized"
-                exit
             fi
+        elif [[ `dcmdump --search 0008,0070 ${first_dcm} 2>/dev/null` == *SIEMENS* ]]; then
+            echo "Replacing Siemens bvals and bvecs"
+            cp `dirname $0`/ABCD_Release_2.0_Diffusion_Tables/Siemens_bvals.txt ${orig_bval}
+            cp `dirname $0`/ABCD_Release_2.0_Diffusion_Tables/Siemens_bvecs.txt ${orig_bvec}
+        elif [[ `dcmdump --search 0008,0070 ${first_dcm} 2>/dev/null` == *PHILIPS* ]]; then
```

Comment:
```suggestion
        elif [[ `dcmdump --search 0008,0070 ${first_dcm} 2>/dev/null` == *Philips* ]]; then
```

It isn't all uppercase in the case of Philips.